design/capm3-remediation-controller-proposal.md

```diff
@@ -143,15 +148,16 @@ metadata:
   namespace: NAMESPACE_OF_UNHEALTHY_MACHINE
   ownerReferences:
    - apiVersion: cluster.x-k8s.io/v1alphaX
-     kind: MachineHealthCheck
-     name: REMEDIATION_GROUP
+     kind: Machine
+     name: MACHINE_NAME
 spec:
   type: reboot
```

Comment:
An enumeration of all the `spec.type`  values that we plan to implement would be a good idea.